SU-HSDO/suhumsci
# shs-890 - apply the fixed image/text height to mobile

## Summary
This adjusts mobile/tablet hero slider carousels to have fixed heights similar to how it is already employed on desktop.
This is accomplished by making mobile/tablet images `object-fit: cover` and by removing the screen size check in the pre-existing JS that applies inline styles. 

## Steps to Test
1. After pulling down this page, create a flexible page.
2. Add a `Hero (Layered) - Slider` to the Full-width region. Add two or three slides with different image aspect ratios and with different text lengths.
3. Save and scroll through the slider at mobile to observe that: the image takes the same height and the box container for the title/text takes the same height across slides.
4. Refresh and do the same for desktop as due diligence to make sure it doesn't break pre-existing functionality. 

## Screenshots
![Screen Shot 2022-08-09 at 2 05 09 PM](https://user-images.githubusercontent.com/8405274/183727186-e180437b-d65e-4767-8de1-a2c111b92db9.png)
